README.md

```diff
@@ -1,12 +1,17 @@
-# RunBook
-
-Student Robotics Competition Program
+# Student Robotics Volunteer RunBook
 
 [![CircleCI](https://circleci.com/gh/srobo/runbook.svg?style=svg)](https://circleci.com/gh/srobo/runbook)
 
-# Installation
+The runbook aims to be a single source for all Student Robotics internal
+documentation. This includes information for the [Competition Team][competition-team]
+and [Kit Team][kit-team], as well as other activities.
+
+[competition-team]: https://opsmanual.studentrobotics.org/annual-robotics-competition/competition-programme-team
+[kit-team]: https://opsmanual.studentrobotics.org/v/master/annual-robotics-competition/kit-team
```

Comment:
Worth noting that this is not the final revision and it is probably ill-advised to link to draft versions (`v/master`) of the Ops manual.

Reply:
Yeah, I'm not sure how best to otherwise link to it though? Is there a way to link to a specific revision (git hash, not published version)?

Reply:
(An alternative might be to just not link to the kit team for now)

Reply:
It won't be finalised until at least Wednesday, and that would be if the kit team committee are happy with it in it's current state.

I guess it depends on how soon you want this PR merged.